libs/blocks/chart/chart.js

```diff
@@ -115,6 +113,17 @@ export const getChartOptions = (chartType, dataset, colors, unit = '') => {
   };
 };
 
+const handleIntersect = (chart, chartOptions) => (entries, observer) => {
+  if (!isType(/Array/g, entries)) return;
```

Comment:
nit: I prefer `Array.isArray` but either way works

Reply:
A lot of the type checking came from the previous implementation, I'm not sure if one is safer / better than the other. @meganthecoder do you have any insights on this?

Reply:
I think they're equivalent in terms of safety. The only benefit I see to using `Array.isArray` is that it means one less import